src/compute/src/render/reduce.rs

```diff
@@ -1403,25 +1407,16 @@ where
     };
 
     let debug_name = debug_name.to_string();
-    collection
+    let debug_full_aggrs = full_aggrs.clone();
```

Comment:
Why is this variable called "debug_"? I understand that for `debug_name` it's because the name is intended for debug logging, but that's not the case for the `full_aggrs` clone used in error checking, right?

Reply:
Agreed, I tried to improve the naming here to `err_full_aggrs`.